src/zzz_od/application/notify/notify_app.py

```
@@ -55,13 +58,17 @@ def format_message(self) -> str:
                 continue
             if not self.is_within_time(run_record.run_time):
                 continue
+            if app_config.app_id == charge_plan_const.APP_ID:
+                charge_power_text = self.get_charge_power_text(run_record)
             if run_record.run_status_under_now == AppRunRecord.STATUS_SUCCESS:
                 success.append(app_config.app_name)
             if run_record.run_status_under_now == AppRunRecord.STATUS_FAIL:
                 failure.append(app_config.app_name)
                 self.exist_failure = True
 
         parts = [f"一条龙运行完成："]
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)